moyarsuccess/gathering
# T25 add verification to sign up page

using Firebase to verify user:
every time a user signs up, firebase will email the user with a verification link. user should verify, get back to the app and be able to sign in/go to the home screen from the verification page. 


- create UI for the verification page
- add ui in navHost with the label and its action
- add verificationFragment
- add verificationViewModel
- add verificationViewState
- navigate the user from the verification page to the home screen
